app/web/package.json

```diff
@@ -67,6 +66,8 @@
   },
   "devDependencies": {
     "@babel/core": "^7.26.9",
+    "@next/bundle-analyzer": "^15.4.0-canary.51",
```

Comment:
Isn't a canary a beta version?

Reply:
I’m not sure, this installed with yarn add. It’s just a dev dep tho